zrml/prediction-markets/src/migrations.rs

```diff
@@ -17,7 +17,7 @@ pub mod _0_1_1_move_storage_to_simple_disputes_and_market_commons {
     const MARKET_IDS_PER_REPORT_BLOCK: &[u8] = b"MarketIdsPerReportBlock";
     const MARKET_TO_SWAP_POOL: &[u8] = b"MarketToSwapPool";
     const MARKETS: &[u8] = b"Markets";
-    const PM: &[u8] = b"PredictionMarket";
+    const PM: &[u8] = b"PredictionMarkets";
```

Comment:
Runtime uses `PredictionMarkets` instead of `PredictionMarket`, thus the reason why migrations weren't working.
![Capture d’écran de 2021-07-01 11-11-55](https://user-images.githubusercontent.com/17877264/124138941-8601a300-da5d-11eb-83fc-a2a865d6dbb9.png)